content/en/project/fMRI_ADHD_connectivity/index.md

```diff
@@ -0,0 +1,199 @@
+---
+type: "project" # DON'T TOUCH THIS ! :)
+date: "2023-06-09" # Date you first upload your project.
+# Title of your project (we like creative title) <!-- changed here -->
+title: Analyzing variability of working memory and reward processing in children with and without ADHD using fMRI data
+
+# List the names of the collaborators within the [ ]. If alone, simple put your name within [] <!-- changed here -->
+names: [Zara Khan, Fariah Sandhu, Clara Sun]
+
+# Your project GitHub repository URL <!-- this is changed too -->
+github_repo: https://github.com/brainhack-school2023/csun_project/
+
+# If you are working on a project that has website, indicate the full url including "https://" below or leave it empty. <!-- changed here too -->
+website:
+
+# List +- 4 keywords that best describe your project within []. Note that the project summary also involves a number of key words. Those are listed on top of the [github repository](https://github.com/brainhack-school2020/project_template), click `manage topics`. <!-- changed here as well -->
+# Please only lowercase letters
+tags: [project, github, markdown, brainhack, jupyter notebook, preprocessing, ADHD, fMRI]
+
+# Summarize your project in < ~75 words. This description will appear at the top of your page and on the list page with other projects.. <!-- this is changed as well -->
+
+summary: "The focus of our project was to gain experience using neuroimaging tools to preprocess, analyze, and visualize functional MRI data. We aimed to explore differential variability in brain connectivity among children with and without ADHD. Project reports are incorporated on the BHS [website](https://school.brainhackmtl.org/project)."
+
+# If you want to add a cover image (listpage and image in the right), add it to your directory and indicate the name
+# below with the extension.
+image: "bhs2020.png" 
```

Comment:
Please add a custom cover image for your project.

Reply:
Thank you for the feedback and suggestion Pierre! We have now added a custom cover image. Great to see so many incredible projects starting to show up on the Brainhack School website!